ble_controller/doc/README.rst

```diff
@@ -0,0 +1,116 @@
+| nRF52840   |        |        | X      |
++------------+--------+--------+--------+
+
+Feature support:
+
++--------------------------+--------+--------+--------+
+|                          | s112   | s132   | s140   |
++==========================+========+========+========+
+| 2 Mbps PHY               | X      | X      | X      |
++--------------------------+--------+--------+--------+
+| Advertiser               | X      | X      | X      |
++--------------------------+--------+--------+--------+
+| Slave                    | X      | X      | X      |
++--------------------------+--------+--------+--------+
+| Scanner                  |        | X      | X      |
++--------------------------+--------+--------+--------+
+| Master                   |        | X      | X      |
++--------------------------+--------+--------+--------+
+| Data Length Extensions   |        | X      | X      |
++--------------------------+--------+--------+--------+
+| Advertising Extensions   |        | X      | X      |
++--------------------------+--------+--------+--------+
+| Coded PHY (Long Range)   |        |        | X      |
++--------------------------+--------+--------+--------+
+
+*Disclaimer*
+
+-  The libraries are for evaluation purposes only.
+-  The libraries are not fully functional and not yet fully tested.
+-  The libraries are not optimized for performance.
+
+Integration guide
+-----------------
+
+Applications utilizing the BLE Controller library are responsible for
+the following:
+
+-  Forward RTC0, TIMER0, RADIO, POWER\_CLOCK, and RNG interrupts to the
+   BLE Controller library.
+   The application should not add additional processing in the
+   corresponding interrupt handlers,
+   otherwise the behavior is undefined.
+-  Call a signal handler when the low priority task processing interrupt occurs.
+-  Ensure thread safe operation. This is achieved by either:
+
+   -  Call all APIs from the same execution priority.
+   -  Ensure that no API call is interrupted by other API calls, i.e. by
+      using critical sections.
+
+-  Configure the number of available links:
+
+   -  Link configuration must be done before enabling the controller.
+   -  The application is required to provide a memory buffer the size of
+      which is dependent on the link configuration.
+
+The application is not allowed to access the following peripherals
+directly:
+
+-  RTC0
+-  TIMER0
+-  RADIO
+-  CLOCK
+-  RNG
+-  TEMP
+-  ECB
+-  AAR
+-  NVMC
+-  PPI channels 17 - 31
+
+In addition to these peripherals, the BLE Controller library requires one
+interrupt for low priority task processing. This interrupt is provided by the
+application in ``ble_controller_init()``.
+
+The APIs provided in ble\_controller\_soc.h and timeslot.h gives the
+application limited access to these peripherals.
+
+Architecture diagrams
+---------------------
+
+The image below shows how the BLE Controller library integrates in an
+RTOS-free environment.
+
+|image0|
```

Comment:
the images don't work for me when I build on Windows, it's clearly a slash/backslash issue:
\ble_controller\doc\README.rst:118: WARNING: image file not readable: ble_controller\doc\pic/Architecture_With_RTOS.svg

I don't even know if this will solve the issue, but this is the way we usually include images in RST (example):
.. figure:: images/nrf9160_ug_overview.svg
   :alt: Overview of nRF91 application architecture

Reply:
Should be fixed now.